BfA/SiegeOfBoralus/Trash.lua

```diff
@@ -207,6 +217,7 @@ end
 function mod:KulTiranHalberdEngaged(guid)
 	self:Nameplate(256627, 7.1, guid) -- Slobber Knocker
 	self:Nameplate(257732, 13.1, guid) -- Shattering Bellow
+	self:CDBar(257732, 13.1) -- Shattering Bellow
```

Comment:
In general we can't use CDBars like this because:

1. Most trash mobs are not unique. It's possible to pull more than one, and then the timer is just junk data as it will overwrite itself.
2. CDBars persist until cleared. The `:Death` callback is not enough. For example you could be out of range of the `UNIT_DIED` event that triggers it, or a Hunter could pull this mob and Feign, etc. Any of these situations would result in the bar being stuck on your screen for the rest of the dungeon.

It is possible to overcome 2, but it's not trivial to overcome 1. An idea I've been intending to explore soon would be an option to redirect or copy certain `:Nameplate` timers to the standard bar area in a way that avoids both 1 and 2. But that would be done within BigWigs itself, without needing to go and add extra code to every module.

So for now please leave out the `CDBar` additions to this file.

Reply:
Alright, I'll remove all CDBars and submit the commit. However, it's really tough to spot the "AOE in 5 seconds" alert among a slew of nameplates. Displaying trash mobs' AOE abilities separately on the CDBar is incredibly helpful.